.github/workflows/test-push-plugins.yaml

```diff
@@ -0,0 +1,54 @@
+name: Publish plugins
+on:
+  push:
+    branches:
+    - main
+    paths:
+    - 'cmd/executor/**'
+    - 'cmd/source/**'
+jobs:
+  publish_plugins:
+    permissions:
+      contents: 'read'
+      id-token: 'write'
+    runs-on: ubuntu-latest
+    env:
+      GO111MODULE: on
+      GOPATH: /home/runner/work/botkube
+      GOBIN: /home/runner/work/botkube/bin
+      DOCKER_CLI_EXPERIMENTAL: "enabled"
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+    - name: GCP auth
+      uses: 'google-github-actions/auth@v1'
+      with:
+        credentials_json: ${{ secrets.PLUGINS_BUCKET_CREDENTIALS }}
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v1
+    - name: Setup Go
+      uses: actions/setup-go@v3
+      with:
+        go-version-file: 'go.mod'
+        cache: true
+    - name: Install GoReleaser
+      uses: goreleaser/goreleaser-action@v2
+      with:
+        install-only: true
+        version: latest
+    - name: Build all plugins into dist directory
+      env:
+        GORELEASER_CURRENT_TAG: "v1.0.0-devel"
```

Comment:
🤔 I know we used this for integration tests, but this is different case, because the version is public. Can we change this `v9.99.9-dev`? It's not that I like `v9.99.9-dev` tag, but just for consistency reason it would be better than 1.0.0. Somebody can think we're going to release 1.0.0 soon 😄 

Other ideas: `v0.0.0-latest`, `v0.0.0-dev`... but probably consistency is better.

What do you think?

PS you can also change the version for the integration tests.

Reply:
I there any place in the test where the version is also hard-coded?

Reply:
Yes, branch build, but you already changed 👍